src/components/dateTimePickerInput.js

```diff
@@ -172,11 +176,16 @@
     let format;
     let resultString;
     let use24HourClock = true;
+    const validation = {};
 
     switch (type) {
       case 'date': {
         DateTimeComponent = KeyboardDatePicker;
         format = dateFormat || 'dd/MM/yyyy';
+        if (minValueText)
+          validation.minDate = DateFns.parse(minValueText, dateFormat);
```

Comment:
use let instead of const when modifying vars

Reply:
This is not a variable (re)assignment but a key-value pair addition to an object instead. How should I use "let" here?

Reply:
I looked at this before my Monday morning coffee :stuck_out_tongue: Approved